### PR TITLE
feat: add hotkeys for undo redo and remove buttons

### PR DESCRIPTION
**Pull request by atarashansky**
Because we removed the undo/redo buttons to make room for the embedding menu in the menubar (#466), we added hotkeys to achieve the same functionality.

CTRL+Z --> Undo
CTRL+SHIFT+Z --> Redo

Smoke tests using undo/redo were updated to emulate the hotkey combinations.

### CODE REVIEW

File: client/__tests__/e2e/cellxgeneActions.ts [hunk 1]
```diff
@@ -43,6 +43,20 @@ export async function drag(testId: any, start: any, end: any, lasso = false) {
   await page.mouse.up();
 }
 
+export async function keyboardUndo() {
+  await page.keyboard.down("ControlLeft");
```
[the remark this answers is not in the document]
Comment (atarashansky) [reply client/__tests__/e2e/cellxgeneActions.ts:47]: It doesn't - it'll always be Ctrl. Do you know how to detect if mac and switch to command?

[the remark this answers is not in the document]
Comment (ashin-czi) [reply client/__tests__/e2e/cellxgeneActions.ts:47]: Could check the [useragent for Macs](https://www.whatismybrowser.com/guides/the-latest-user-agent/macos) and use the "MetaLeft" key according to this doc: https://www.w3.org/TR/uievents-code/#key-alphanumeric-functional

[the remark this answers is not in the document]
Comment (atarashansky) [reply client/__tests__/e2e/cellxgeneActions.ts:47]: Just added a fix for this - remains to be seen if puppeteer thinks it's on a mac or windows PC...

[the remark this answers is not in the document]
Comment (atarashansky) [reply client/__tests__/e2e/cellxgeneActions.ts]: seems to think it's on a mac - updated test to use MetaLeft

**Review by seve — APPROVED** [review]

**Pull request merged**